Gems/ScriptCanvas/Code/Include/ScriptCanvas/Core/Node.cpp

```diff
@@ -45,6 +45,9 @@ namespace NodeCpp
         MergeFromBackend2dotZero = 12,
         AddDisabledFlag = 13,
         AddName = 1,
+        // AddName was a version backward, causing errors when processing assets on versions 12 and 13.
```

Comment:
I've updated the comment. I'm trying to capture two things here:
1. The serialization system emits an error when versions go backward, because in most cases it's a data and code mistake - someone updated data and had a local change to versioning they didn't push. It doesn't apply here, but I think the error should remain the same, we should require versions only go up.
2. This node system has a data conversion feature between versions. Because this new version of 2 is actually a conflict with a previous version of 2 from several years ago, before the enum was created, it's going to cause other problems in the future as new versions are added that conflict with previous pre-enum versions. Version conversion will trigger that expects data to look like the old version of a version instead of the new version of a version.